docs/configuration.md

```diff
@@ -1141,6 +1141,48 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.blacklist.enabled</code></td>
+  <td>true</td>
+  <td>
+    If set to ture, executor blacklist feature will be enabled to avoid allocte new task on bad executor. The logic to define bad executor is based on BlacklistStrategy which is also configurable.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.scheduler.executorTaskBlacklistTime</code></td>
+  <td>0L</td>
+  <td>
+    The threshold to deside blacklist executor, if last failed time is older than current time subtract executorTaskBlacklistTime, then the executor could be removed from blacklist.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.scheduler.blacklist.recoverPeriod</code></td>
+  <td>60L</td>
+  <td>
+    The period between runnning blacklist recover process. 
```

Comment:
lets include a timeunit here too, so "60s", and change the text to "If executor blacklisting is enabled, this controls how often to check if executors can be returned to the pool of active executors."